src/site/site.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
@hboutemy It looks like Apache RAT chokes on this file
```
[INFO] --- apache-rat-plugin:0.13:check (default) @ httpclient5-parent ---
[INFO] Enabled default license matchers.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Parsing exclusions from /home/travis/build/apache/httpcomponents-client/.gitignore
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 76 implicit excludes (use -debug for more details).
[INFO] 9 explicit excludes (use -debug for more details).
[INFO] 16 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 1, unknown: 1, generated: 0, approved: 8 licenses.
[INFO] ------------------------------------------------------------------------
```

Reply:
license header updated: now rat is ok